packages/lesswrong/lib/collections/users/callbacks.js

```diff
@@ -16,3 +16,13 @@ function updateTrustedStatus ({newDocument, vote}) {
 
 addCallback("votes.smallUpvote.async", updateTrustedStatus);
 addCallback("votes.bigUpvote.async", updateTrustedStatus);
+
+
+function maybeSendVerificationEmail (user, oldUser)
+{
+  if((""+oldUser.whenConfirmationEmailSent) != (""+user.whenConfirmationEmailSent)) {
```

Comment:
Compare with .getTime, see here: 

https://stackoverflow.com/questions/492994/compare-two-dates-with-javascript

Reply:
Ended up obsoleted by switching to a different trigger, which either provides a $set.whenConfirmationEmailSent or doesn't, so it's no longer a date comparison at all.